ansible_base/authentication/utils/claims.py

```diff
@@ -346,52 +346,47 @@ def create_organizations_and_teams(results) -> None:
     Use the results data from 'create_claims' to make the Organization
     and Team objects necessary for the user if they do not exist.
     """
-
     # a flat list of relevant org names
-    org_list = set()
+    org_set = set()
     # a flat list of relevant org:team names
-    team_list = set()
-
+    team_org_set = set()
     # a structure for caching org+team,member info
     membership_map = {}
 
     # fill in the top level org membership data ...
     for org_name, is_member in results['claims']['organization_membership'].items():
-        if is_member:
-            org_list.add(org_name)
-            membership_map[org_name] = {'id': None, 'teams': {}}
+        if not is_member:
+            continue
+        org_set.add(org_name)
+        membership_map[org_name] = {'id': None, 'teams': []}
 
     # fill in the team membership data ...
     for org_name, teams in results['claims']['team_membership'].items():
         for team_name, is_member in teams.items():
-            if is_member:
-                org_list.add(org_name)
-                team_list.add((org_name, team_name))
-                if org_name not in membership_map:
-                    membership_map[org_name] = {'id': None, 'teams': {}}
-                membership_map[org_name]['teams'][team_name] = True
-
-    # make a map or org name to org id to reduce calls and data sent over the wire
-    existing_orgs = {org.name: org.id for org in Organization.objects.filter(name__in=org_list)}
-
-    # make each org as necessary or simply store the id
-    for org_name in org_list:
-        if org_name not in existing_orgs:
-            new_org, _ = Organization.objects.get_or_create(name=org_name)
-            membership_map[org_name]['id'] = new_org.id
-        else:
-            membership_map[org_name]['id'] = existing_orgs[org_name]
-
+            if not is_member:
```

Comment:
the same like above. Negative condition doesn't improve readability for few lines of code